sigstore/_cli.py

```diff
@@ -122,9 +119,14 @@ def _set_default_verify_subparser(parser: argparse.ArgumentParser, name: str) ->
 def _add_shared_instance_options(group: argparse._ArgumentGroup) -> None:
```

Comment:
N.B.: Conceptually these common options belong at the global `sigstore` level, rather than embedded into each subcommand. I've moved one of them in this PR (`--staging`) because I needed to fix its behavior with `get-identity-token`, but I'll move the rest in a follow-up.